subsys/bluetooth/host/conn.c

```diff
@@ -339,10 +339,12 @@ void bt_conn_reset_rx_state(struct bt_conn *conn)
 	conn->rx = NULL;
 }
 
-static void bt_acl_recv(struct bt_conn *conn, struct net_buf *buf,
-			uint8_t flags)
+static void bt_acl_recv_(struct bt_conn *conn, struct net_buf *buf, uint8_t flags)
 {
 	uint16_t acl_total_len;
+	enum acl_data_flags *buf_flags = bt_get_acl_buf_flags(buf);
+
+	*buf_flags = 0;
 
 	/* Check packet boundary flags */
 	switch (flags) {
```

Comment:
It gets a bit confusing to have both `buf_flags` and `flags` in this function. What type of flags are the latter? 

And should `buf_flags` have been `acl_flags`?

Reply:
it is confusing indeed. I made a function.